packages/deployment/upgrade-test/upgrade-test-scripts/cliHelper.mjs

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable @jessie.js/safe-await-separator */
+import { $, execaCommand } from 'execa';
+import { BINARY } from './constants.mjs';
+
+export const executeCommand = async (command, params, options = {}) => {
+  const { stdout } = await execaCommand(
+    `${command} ${params.join(' ')}`,
+    options,
+  );
+  return stdout;
+};
+
+export const agd = {
+  query: async (...params) => {
+    const newParams = ['query', ...params, '-o json'];
+    const data = await executeCommand(BINARY, newParams);
+    return JSON.parse(data);
+  },
+  tx: async (...params) => {
+    const newParams = ['tx', ...params, '-o json'];
+    const data = await executeCommand(BINARY, newParams, { shell: true });
+    return JSON.parse(data);
+  },
+  keys: async (...params) => {
+    let newParams = ['keys', ...params];
+    let shouldParse = true;
+
+    if (params.includes('show')) {
+      if (params.includes('-a') || params.includes('-address')) {
+        shouldParse = false;
+      }
+    }
+
+    if (shouldParse) {
+      newParams = [...newParams, '--output json'];
+    }
+
+    const data = await executeCommand(BINARY, newParams, { input: 'Y' });
+    if (!shouldParse) {
+      return data;
+    }
+
+    return JSON.parse(data);
+  },
+  export: async (...params) => {
+    const newParams = ['export', ...params];
+    const data = await executeCommand(BINARY, newParams);
+    return JSON.parse(data);
+  },
+};
+
+export const agoric = {
+  follow: async (...params) => {
+    let newParams = ['follow', ...params];
+    let parseJson = false;
+
+    if (!params.includes('-o')) {
+      newParams = [...newParams, '-o json'];
+      parseJson = true;
+    }
+    const data = await executeCommand('agoric', newParams);
+
+    if (parseJson) {
+      return JSON.parse(data);
+    }
+
+    return data;
+  },
+  wallet: async (...params) => {
+    const newParams = ['wallet', ...params];
+    return executeCommand('agoric', newParams);
+  },
+};
+
+export const { stdout: agopsLocation } = await $({
+  shell: true,
+  cwd: '/usr/src/agoric-sdk',
+})`yarn bin agops`;
+
+export const agops = {
```

Comment:
remark: I worry we're creating more work for ourselves by going from JS to shell and shell back to JS. This test is of _upgrade_ . We don't need to integrate with Commander.js. We might better off importing the modules from `agoric-cli` directly. We'd get type checking that way!